.gitignore

```diff
@@ -1,8 +1,176 @@
-*/node_modules/
-/development/.ipfs
+# Common #
+##########
+.env
+.idea
+.sass-cache
+build/
+build_webpack/
+coverage
+coverage.json
+npm-debug.log*
+node_modules/
```

Comment:
There shouldn't be any node_modules getting created in the root dir of the repo ?